docs/documentation/updating-the-kit.md

```diff
@@ -18,13 +18,13 @@ Tell us as much as you can about the issue you're having, and the computer and o
 In Finder on Mac or Windows Explorer go to your prototype folder and open the file `VERSION.txt`. This will show what version of the prototype kit you’re using.
 
 - If your prototype is version 8 or before, then contact the GOV.UK Prototype team for help with updating it
-- If your prototype is version 9 or 10, you can update to version 11 by following the steps on this page
+- If your prototype is version 9, 10 or 11, you can update to version 12 by following the steps on this page
```

Comment:
To avoid needing to remember to come back and update this when future major versions are released, could we generalise it?

```suggestion
- If your prototype is version 9 or above, you can update to the latest version by following the steps on this page
```

Reply:
Perhaps being a little pedantic, but these changes also don't really relate to updating to Frontend v4.0 – they're related to releasing the next major version of the kit, which isn't happening in this PR?

Reply:
We user tested this content and made the changes to naming versions as a result for clarity.

Reply:
Yeah, that's fair - I can pull this commit out into another PR.